openduck-py/openduck_py/routers/voice.py

```diff
@@ -394,5 +400,6 @@ async def audio_response(
 
     # TODO(zach): We never actually close it right now, we wait for the client
     # to close. But we should close it based on some timeout.
+    await consumer_task.join()
```

Comment:
Does this actually work? Since `consumer` runs forever, I would expect this line to also run forever.

Reply:
oh, I might want to do `await responder.response_queue.join()` instead. I think that stops once the queue is empty?

Reply:
like maybe we need a `consumer_task.cancel()` in the `finally` block or something... not sure.